integrations/extensions/docs/elasticsearch-install-and-setup/text_embedding_deploy_and_use.md

```diff
@@ -0,0 +1,184 @@
+# How to set up and use 3rd-party text embeddings for dense vector search in Elasticsearch
+This guide demonstrates how to deploy and use a text embedding model in Elasticsearch. The model will generate vector representations for text, enabling vector similarity (k-nearest neighbours) search.
+
+Please refer to [this guide](./ICD_Elasticsearch_install_and_setup.md) first to create an Elasticsearch instance and set up Kibana if you haven't already.
+
+## Install the eland library
+Run the command below to install the [eland](https://github.com/elastic/eland) library.
+```bash
+$ python -m pip install "eland[pytorch]"
+```
+This library allows us to pull and deploy a 3rd-party text embedding model to our Elasticsearch instance.
```

Comment:
Charles Quincy and I worked together on the following IP warning for 3rd party embedding models.  We are both comfortable with this wording:

> CAUTION: Open source and 3rd party models are not in scope of IBM or Elastic indemnity clauses. Customers must accept relevant terms and conditions to choose or bring their own models. Additionally, IBM has not assessed Elastic's supported multi-lingual models so any use of Elastic-supported models should be understood thoroughly both respect to the terms of use for those models and the terms of use of all of the data that was used to train those models.

Reply:
Perfect thanks. I'm assuming `understood thoroughly both respect to the terms` is `understood thoroughly both with respect to the terms`

Reply:
Right, good catch -- yes, please include that fix.